handbook/business-operations/README.md

```diff
@@ -86,7 +86,7 @@ Here's how:
 
 1. For each hourly core team member in Gusto or Plane.com, find the DRI by checking [who they report to](https://docs.google.com/spreadsheets/d/1OSLn-ZCbGSjPusHPiR5dwQhheH1K8-xqyZdsOe9y7qc/edit#gid=0).
    - If any direct report is hourly in Plane.com and submits hours monthly, still list them and provide an explanation.
-2. Consultants submit their hours through Gusto (US consultants) or Plane.com (international consultants) and require DRI approval. Find the DRI using the [Business Operations KPIs](https://docs.google.com/spreadsheets/d/1Hso0LxqwrRVINCyW_n436bNHmoqhoLhC8bcbvLPOs9A/edit#gid=0).
+2. Consultants submit their hours through Gusto (US consultants) or Plane.com (international consultants) and ree DRI approval. Find the DRI using the [Business Operations KPIs](https://docs.google.com/spreadsheets/d/1Hso0LxqwrRVINCyW_n436bNHmoqhoLhC8bcbvLPOs9A/edit#gid=0).
```

Comment:
Hey @JoStableford, I think you might have some unintentional changes here. "and ree"

Reply:
@Sampfluger88 the "quirk" was using the find function to get to the section I wanted to edit by finding "quir" for quirk, but ended up unintentionally replacing with " " instead.. putting the "quir" in quirk!